<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>70</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>89</red>
        <green>62</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>70</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>107</red>
        <green>118</green>
        <blue>139</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFunction_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1001</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gilroy</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;НАХОЖДЕНИЕ КОРНЯ ФУНКЦИИ &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>436</width>
      <height>283</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFunction">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Функция f(x):     </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFunction">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelInterval">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Отрезок [a, b]:   </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditInterval">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxMethod">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelMaxIterations">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Число итераций:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelTolerance">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Точность:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelMethod">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Метод:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditMaxIterations">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditTolerance">
         <property name="font">
          <font>
           <family>Gilroy</family>
           <pointsize>22</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSolve">
       <property name="font">
        <font>
         <family>Gilroy</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>РЕШИТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;title&gt;Методичка по вводу данных&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#f9f9f9&quot;&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:large; font-weight:700; color:#4caf50;&quot;&gt;Методичка по вводу данных&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:large; font-weight:700; color:#333333;&quot;&gt; &lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; font-weight:700; color:#d32f2f;&quot;&gt;Функция:&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt; Укажите функцию, которая будет решаться на интервале. Функция может содержать латинские буквы, цифры, математические операторы и скобки.&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:12px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Пример:&lt;/span&gt; &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;x^2 - 4&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;cos(x)&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;3x + 2&lt;/span&gt; &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#ff5722;&quot;&gt;Важно:&lt;/span&gt; Не используйте русские буквы или специальные символы, такие как &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;@, $, #&lt;/span&gt;. &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Недопустимо:&lt;/span&gt; Использовать пробелы между переменной и оператором, например, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;x + 2&lt;/span&gt;. &lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; font-weight:700; color:#d32f2f;&quot;&gt;Интервал:&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt; Укажите интервал, на котором будет решаться уравнение.&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:12px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Пример:&lt;/span&gt; &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;2, 3&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;1.5, 4.7&lt;/span&gt; &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#ff5722;&quot;&gt;Важно:&lt;/span&gt; Интервал должен быть записан в формате, например, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;a, b&lt;/span&gt;, где a и b — это числа. &lt;br /&gt;Не используйте квадратные скобки, такие как &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;[2, 3]&lt;/span&gt;. &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Недопустимо:&lt;/span&gt; Использовать пробелы внутри чисел, или писать интервал как &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;(2,3)&lt;/span&gt; или &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;2 3&lt;/span&gt;. &lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; font-weight:700; color:#d32f2f;&quot;&gt;Точность:&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt; Укажите точность, с которой будет найден корень уравнения.&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:12px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Пример:&lt;/span&gt; &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.001&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.01&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.0001&lt;/span&gt; &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#ff5722;&quot;&gt;Важно:&lt;/span&gt; Точность должна быть положительным числом, например, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.001&lt;/span&gt; или &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.01&lt;/span&gt;. &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Недопустимо:&lt;/span&gt; Указывать точность больше &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;0.1&lt;/span&gt;, а также использовать ноль или отрицательные числа. &lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; font-weight:700; color:#d32f2f;&quot;&gt;Число итераций:&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt; Укажите количество шагов, которое будет выполнено для нахождения корня.&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:12px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Пример:&lt;/span&gt; &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;50&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;100&lt;/span&gt;, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;10&lt;/span&gt; &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:10px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#ff5722;&quot;&gt;Важно:&lt;/span&gt; Количество итераций должно быть целым числом в пределах от &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;1&lt;/span&gt; до &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;100&lt;/span&gt;. &lt;/li&gt;
&lt;li style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot; style=&quot; margin-top:10px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:160%;&quot;&gt;&lt;span style=&quot; font-weight:700; color:#d32f2f;&quot;&gt;Недопустимо:&lt;/span&gt; Указывать отрицательные числа или больше 100. Например, &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;-10&lt;/span&gt; или &lt;span style=&quot; font-style:italic; color:#2196f3;&quot;&gt;150&lt;/span&gt; — ошибка. &lt;/li&gt;&lt;/ul&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt;При вводе функции используйте только латинские буквы, цифры и математические операторы, такие как &lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; font-style:italic; color:#2196f3;&quot;&gt;+, -, *, /, ^&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; font-size:14px; color:#333333;&quot;&gt;.&lt;/span&gt;&lt;span style=&quot; font-family:'Arial','sans-serif'; color:#333333;&quot;&gt; &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
